### PR TITLE
fix(llamaindex): support v0.1.0

### PR DESCRIPTION
**Pull request by nirga**

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "version": "0.5.3",
+  "version": "0.5.4-alpha.0",
```
Comment (galkleinman) [inline lerna.json:3]: remove the alpha?

File: packages/instrumentation-langchain/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@traceloop/instrumentation-langchain",
-  "version": "0.5.0",
+  "version": "0.5.4-alpha.0",
```
Comment (galkleinman) [inline packages/instrumentation-langchain/package.json:3]: same

File: packages/instrumentation-llamaindex/src/custom-llm-instrumentation.ts [hunk 3]
```diff
@@ -166,45 +107,50 @@ export class CustomLLMInstrumentation {
     };
   }
 
-  handleResponse(
-    result: any,
+  handleResponse<T extends ResponseType>(
```
Comment (galkleinman) [inline packages/instrumentation-llamaindex/src/custom-llm-instrumentation.ts:110]: it should be `extends` or there is some notion of `is` or something like that?

**Review by galkleinman — APPROVED** [review]
LGTM

File: packages/instrumentation-llamaindex/src/custom-llm-instrumentation.ts (hunk 3 above)
Comment (nirga) [reply packages/instrumentation-llamaindex/src/custom-llm-instrumentation.ts:110]: No notion of `is` unfortunately

File: lerna.json (hunk 1 above)
Comment (nirga) [reply lerna.json:3]: This will get auto updated by lerna when I release

File: packages/instrumentation-langchain/package.json (hunk 2 above)
Comment (nirga) [reply packages/instrumentation-langchain/package.json:3]: This will get auto updated by lerna when I release

**Pull request merged**